test/Libraries/WorkflowTests/DynamoSamples.cs

```diff
@@ -536,9 +536,14 @@ public void ImportExport_Data_To_Excel()
             AssertPreviewCount(lineNodeID, 65);
 
             // Killing excel process if there is any after running the graph.
-            Process[] procs = Process.GetProcessesByName("excel");
-            foreach (Process proc in procs)
-                proc.Kill();
+            foreach (var process in Process.GetProcessesByName("EXCEL"))
+            {
+                if (process.MainWindowTitle.Equals("ImportExport_Data To Excel - Excel"))
```

Comment:
What process is it that does not have this title? IOW, what other titles are there? Is this title specific to Dynamo or the tests?

Reply:
The titles are specific to each dyn. If a test runs a dyn that reads/writes an excel file name "test1", then the excel window will have the title "test1 - Excel". This title is set by the excel interop API (AFAIK)